### DISCUSSION TITLE
No valid animation file created with LS-Dyna input file (Solid elements)

### DISCUSSION THREAD

**Discussion by FGarelli** [OpenRadioss/OpenRadioss#1503] [Q&A]
Hi, I didn't manage to receive any output from this simple simulation created with SOLID elements. I cannot find the problem but i guess is related to the engine file and some DT animation file. Does anyone have the same problem or know how to solve it?
In the attachment the file that i'm using:
[extruded_0000.txt](https://github.com/OpenRadioss/OpenRadioss/files/11555090/extruded_0000.txt)
[extruded_0001.rad.txt](https://github.com/OpenRadioss/OpenRadioss/files/11555091/extruded_0001.rad.txt)
[extruded_0001.txt](https://github.com/OpenRadioss/OpenRadioss/files/11555092/extruded_0001.txt)
[extruded.txt](https://github.com/OpenRadioss/OpenRadioss/files/11555093/extruded.txt)

**Comment by sebastienVilleneuve** [comment]
Hi @FGarelli 

In your 1.out file , I can see writing of animation files : 
Example :    ANIMATION FILE: extrudedA013 WRITTEN


Sébastien

**Reply by FGarelli** [reply]
Yes, the software write an animation file which is empty and cannot be visualised in Paraview. Do you have any guess of what the problem can be? I performed many shell simulations without problems, I found this problem just with solid elements.
